src/include/network/lan/wizards.rb

```diff
@@ -44,7 +44,6 @@ def initialize_network_lan_wizards(include_target)
       Yast.include include_target, "network/lan/address.rb"
       Yast.include include_target, "network/lan/complex.rb"
       Yast.include include_target, "network/lan/dhcp.rb"
-      Yast.include include_target, "network/lan/hardware.rb"
```

Comment:
This is wrong, it's a different file.

Reply:
Currently it only contains an s390 activation dialog, maybe we should move it to https://github.com/yast/yast-network/blob/master/src/include/network/lan/s390.rb

Reply:
BTW, thnx for the catch, fixed.